twitter/direct_messages.go

```diff
@@ -14,3 +19,119 @@ type DirectMessage struct {
 	SenderScreenName    string    `json:"sender_screen_name"`
 	Text                string    `json:"text"`
 }
+
+// DirectMessageService provides methods for accessing Twitter status API endpoints.
+type DirectMessageService struct {
+	baseSling *sling.Sling
+	sling     *sling.Sling
+}
+
+// newDirectMessageService returns a new StatusService.
+func newDirectMessageService(sling *sling.Sling) *DirectMessageService {
+	return &DirectMessageService{
+		baseSling: sling.New(),
+		sling:     sling.Path("direct_messages/"),
+	}
+}
+
+// DirectMessageShowParams are the parameters for DirectMessageService.Show
+type DirectMessageShowParams struct {
+	ID int64 `url:"id,omitempty"`
+}
+
+// Show returns the requested DirectMessage and all response messages.
+// https://dev.twitter.com/rest/reference/get/direct_messages/show
+func (s *DirectMessageService) Show(id int64) ([]DirectMessage, *http.Response, error) {
```

Comment:
Show returns a single Direct Message. Parsing as a list doesn't work if you try it.